### DISCUSSION TITLE
nvim-qt maxmize window

### DISCUSSION THREAD

**Discussion by rickywu** [ayamir/nvimdots#386] [General]
nvim-qt alwasy not maxmized after reboot in Windows, I add these in init.lua in core/init.lua
```
	if global.is_windows then
            vim.cmd([[call GuiWindowMaximized(1)]])
	end
```
but got error:
E5113: Error while calling lua chunk: d:\DevTools\Neovim\nvim/lua/core/init.lua:137: Vim(call):E117: Unknown function: GuiWindowMaximized

call GuiWindowMaximized(1) works well after startup, so how to maxmize in lua?

**Comment by ayamir** [comment]
1. New a file named `ginit.vim` in `C:\Users\YourUserName\AppData\Local\nvim`.
2. Add these lines to it:
```vim
if GuiName == 'nvim-qt'
    call GuiWindowMaximized(1)
endif
```
